### ISSUE TITLE
MTP prediction model learns a fixed-angle trajectory

### ISSUE THREAD

**Issue by SSSSER1994** [nutonomy/nuscenes-devkit#540]
I use MTP loss to train MTP model, and set the mode to 3 as the paper recommended.  However, although my training data set has diverse directions of trajectories, the model learns 3 fixed-angle trajectories. **And the feature of the map is somehow lost**. I'm fused how the model can learn the different trajectory direction in one mode(e.g. left) when the loss contains _cross-entropy_ part, which makes _regression_ part less impactful. Is there any notices during training(such as improve epochs?)? Or is anyone have trained a good model?

**Comment by emwolff** [comment]
@SSSSER1994 Could you please provide some more concrete examples of what you are seeing? The CNN encoder of the scene context includes the map info to whatever extent it is captured. I would suggest taking a close look at the original MTP paper: https://arxiv.org/abs/1809.10732.

**Comment by SSSSER1994** [comment]
I use the original image and state vector to train, and omit the normalization process, once I add normalization, the result seems to improve somehow. The feature of map impacts more.
